etna/transforms/embeddings/models/ts2vec.py

```diff
@@ -39,6 +46,7 @@ def __init__(
         num_workers: int = 0,
         max_train_length: Optional[int] = None,
         temporal_unit: int = 0,
+        freezed: bool = False,
```

Comment:
Why not `is_freezed`? It can cause some confusion that somewhere it is `is_freezed`, somewhere it `freezed`.

Reply:
We already have `is_freezed` property.

Reply:
That's why I don't understand why do we need additional `freezed` parameter (ans `self.freezed` also).